build/debian/owasp-zap.desktop

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env xdg-open
```

Comment:
This is what i have found in general desktop files so far and have not tried without this. so that's why keeping it as is.